code/modules/client/client procs.dm

```diff
@@ -157,7 +157,7 @@
 	nanomanager.send_resources(src)
 
 	if(!void)
-		void = new()
+		void = void.MakeGreed()
```

Comment:
You've replaced the line ```void = new()``` while it still needs to be there, as otherwise you're calling null.MakeGreed().